server/endpoints.md

```diff
@@ -177,6 +177,21 @@ Requires login as teacher
   - _response body_
     - success: bool
 
+# Edit a block of lecture online status
+
+Requires login as support officer
+
+- **PATCH** /api/lectures/online
+  - _request params_
+    - value: bool
```

Comment:
Define better name for value, or dedscribe behavior if true/false

Reply:
```suggestion
    - value: bool (true online, false offline)
```

Reply:
```suggestion
    - value: bool (true turn all to online, false turn all to presence)
```